### PR TITLE
Fix: Polygon network names

### PR DESCRIPTION
**Pull request by josemarinas**
## Description

Please include a summary of the change and be sure you follow the contributions rules we do provide [here](./CONTRIBUTIONS.md)

Task: [ID]()

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder of the package after the [UPCOMING] title and before the latest version.
- [ ] I have tested my code on the test network.

### CODE REVIEW

File: modules/client/src/client-common/constants.ts [hunk 1]
```diff
@@ -48,7 +48,7 @@ export const LIVE_CONTRACTS: { [K in SupportedNetworks]: NetworkDeployment } = {
     addresslistVotingSetup: activeContractsList.goerli.AddresslistVotingSetup,
     tokenVotingSetup: activeContractsList.goerli.TokenVotingSetup,
   },
-  mumbai: {
+  maticmum: {
```
Comment (brickpop) [inline modules/client/src/client-common/constants.ts:51]: Maticum?

File: modules/client/src/client-common/interfaces/common.ts [hunk 2]
```diff
@@ -1,4 +1,9 @@
-export const SupportedNetworksArray = ["mainnet", "goerli", "mumbai", "polygon"] as const;
+export const SupportedNetworksArray = [
+  "mainnet",
+  "goerli",
+  "matic",
+  "maticmum",
```
Comment (brickpop) [inline modules/client/src/client-common/interfaces/common.ts:5]: Shouldn't these be `matic` and `mumbai` ?

File: modules/client/src/client-common/interfaces/common.ts [hunk 3]
```diff
@@ -11,6 +16,7 @@ export type NetworkDeployment = {
   adminSetup: string;
   addresslistVotingSetup: string;
   tokenVotingSetup: string;
+  publicResolver?: string;
```
Comment (brickpop) [inline modules/client/src/client-common/interfaces/common.ts]: The variable should include the ENS registry, which we deploy
The resolver is defined at the registry level

File: modules/client/src/client-common/interfaces/common.ts (hunk 2 above)
Comment (josemarinas) [reply modules/client/src/client-common/interfaces/common.ts:5]: nope, the name used by all provider is maticmum

Comment (josemarinas) [reply modules/client/src/client-common/interfaces/common.ts:5]: https://docs.ethers.org/v5/api/providers/api-providers/#EtherscanProvider

**Review by brickpop — APPROVED** [review]

**Pull request merged**